subgraphs/opensea/protocols/opensea-v2/src/constants.ts

```diff
@@ -0,0 +1,6 @@
+export const PROTOCOL_SUBGRAPH_VERSION = "1.0.0";
+export const PROTOCOL_METHODOLOGY_VERSION = "1.0.0";
+export const PROTOCOL_MARKETPLACE_NAME = "OpenSea: Wyvern Exchange v2";
+export const PROTOCOL_MARKETPLACE_SLUG = "opensea-v2";
+export const PROTOCOL_MARKETPLACE_ADDRESS =
+  "0x7f268357A8c2552623316e2562D90e642bB538E5";
```

Comment:
let's try keep all addresses lowercase going forward to avoid unexpected issues

Reply:
a trick could be using `Address` type instead, which is case insensitive

Reply:
i'll just make it lowercase